app/models/session.ts

```diff
@@ -14,32 +14,28 @@ export default class SessionModel extends Model {
   @attr('date') declare startedAt?: Date;
   @attr('date') declare endedAt?: Date;
 
-  @hasMany('agenda-item', { async: true, inverse: 'session' })
+  @hasMany('agenda-item', { async: true, inverse: 'sessions' })
   declare agendaItems: AsyncHasMany<AgendaItemModel>;
 
-  @belongsTo('governing-body', { async: false, inverse: 'session' })
+  @belongsTo('governing-body', { async: false, inverse: 'sessions' })
   declare governingBody: GoverningBodyModel;
 
   get name() {
```

Comment:
**Suggestion:** Should we perhaps change this getters name? It's a bit ambiguously named